src/WebJobs.Extensions.OpenAI.AISearch/OpenAIAzureAISearch.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Microsoft.Azure.Functions.Extensions.DependencyInjection;
+using Microsoft.Azure.WebJobs.Extensions.OpenAI.AzureAISearch;
+using Microsoft.Azure.WebJobs.Extensions.OpenAI.Search;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+
+// Reference: https://docs.microsoft.com/en-us/azure/azure-functions/functions-dotnet-dependency-injection
+[assembly: FunctionsStartup(typeof(OpenAIAzureAISearch))]
+
+namespace Microsoft.Azure.WebJobs.Extensions.OpenAI.AzureAISearch;
+
+class OpenAIAzureAISearch : FunctionsStartup
+{
+    public override void Configure(IFunctionsHostBuilder builder)
+    {
+        builder.Services.AddOptions<AzureAISearchConfigOptions>()
+            .Configure<IConfiguration>((options, config) =>
+            {
+                // For in-proc
```

Comment:
What do you mean by "For in-proc" here? I'd expect this code to execute always.

Reply:
For chat state storage in OOP, Aishwarya has configured another set of options, I will remove the comment for now and let her check it for oop.